tutorials/io/30_reading_fnirs_data.py

```diff
@@ -6,39 +6,78 @@
 Importing data from fNIRS devices
 =================================
 
-MNE includes various functions and utilities for reading NIRS
-data and optode locations.
-
-fNIRS devices consist of light sources and light detectors. A channel is formed
-by source-detector pairs. MNE stores the location of the channels, sources, and
+MNE includes various functions and utilities for reading fNIRS
+data and optode locations. Regardless of the manufacturer and file format,
+MNE stores both the measurement data and metadata in a consistent manner.
+
+fNIRS devices consist of light sources and light detectors, 
+often also termed emitter/transmitter and receiver respectively.
+A channel is formed by source-detector pairs, and MNE represents the
+channel location as the midpoint between source and detector.
+MNE stores the location of the channels, sources, and
 detectors.
+There are a variety of fNIRS data types which can be represented in MNE.
+For continuous wave fNIRS data this includes amplitude, optical density, 
+oxyhaemoglobin, and deoxyhemoglobin.
+And for frequency domain fNIRS this additionally includes
+AC amplitude and phase.
+Different vendors save the data as different data types, and MNE will load
+the data as the appropriate type.
+
+.. warning:: MNE expects a specific formatting of channel names.
+             Manual modification of channel names is not recommended.
+
+
+.. _import-snirf:
+
+SNIRF (.snirf)
+================================
```

Comment:
Sphinx complains about header levels, so I would add a second-level header `***` before this to make sure this `====` lands as a third-level header (also the existing underline is too long):
```suggestion
*****************
Standardized data
*****************

SNIRF (.snirf)
==============
```